difftree/internal/radixmerkle/noder.go

```diff
@@ -0,0 +1,20 @@
+package merkletrie
+
+// The Noder interface is implemented by the elements of a Merkle Trie.
+type Noder interface {
```

Comment:
Yeah, I agree.

I'll probably move it in the next patch, after having used it a little bit for some real work.